ClangSharp/ClangSharp.csproj

```diff
@@ -13,9 +13,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-      <Content Include="ClangSharp.targets" Pack="true" PackagePath="build"/>
-      <Content Include="x64\libclang.dll" Pack="true" PackagePath="content\x64"/>
-      <Content Include="x86\libclang.dll" Pack="true" PackagePath="content\x86"/>
+    <PackageReference Include="libclang.runtime" Version="8.*" />

```

Comment:
`*` versions are generally considered bad practice as it leads to non-deterministic builds. It also requires nuget to always do a server query as the version may have changed.